mkt/comm/models.py

```diff
@@ -134,21 +134,22 @@ def user_has_perm_note(note, profile, request=None):
     Developers of the add-on used in the note, users in the CC list,
     and users who post to the thread are allowed to access the object.
 
-    Moreover, other object permissions are also checked agaisnt the ACLs
+    Moreover, other object permissions are also checked against the ACLs
     of the user.
     """
     if note.author and note.author.id == profile.id:
         # Let the person access their own note.
         return True
 
-    if note.note_type == comm.DEVELOPER_COMMENT:
-        # Developer comment only for developers.
-        return note.thread.check_obj_author(profile)
-
     if request and note.note_type == comm.REVIEWER_COMMENT:
-        # Reviewer comment only for reviewers.
+        # Intenral reviewer comment only for reviewers.
```

Comment:
yeh i saw...was just gonna go fix it whenever i had to address whatever else comments i need to